src/window/index.ts

```diff
@@ -24,6 +24,7 @@ function createWindow(name: string, url: string, prefs: Electron.WebPreferences)
       }
       window.show();
     }
+    window.loadURL(url);
```

Comment:
Why is this needed? I found the only way to trigger this line of code was to pick the `tray: Preferences` menu item to hit the `createWindow` function when the window existed, but it had its content. Is there a circumstance where this line is needed? Should it have its own bug? 

Maybe all this `page` stuff should be written up in a separate issue and have its own PR?

Reply:
Also it's possible that the `will-navigate` appears to happen instantly after `createWindow` finishes that you don't need to load the URL there. But I don't know what the issue is here.

Reply:
Dropped the window focusing stuff from this PR; we can deal with this in a follow-up.

(This was done to match the place where we raise the new error, so that we can automatically focus the K8s page for the user to click (hard) reset since they can't do anything else with the cluster)